tensorflow/core/ops/array_ops.cc

```diff
@@ -410,6 +410,28 @@ axis: Dimension along which to unpack.  Negative values wrap around, so the
 output: The list of tensors unpacked from `value`.
 )doc");
 
+REGISTER_OP("UnravelIndex")
+    .Input("indices: Tidx")
+    .Input("dims: Tidx")
+    .Output("output: Tidx")
+    .Attr("Tidx: {int32, int64} = DT_INT32")
+    .SetShapeFn([](InferenceContext* c) { return Status::OK(); })
+    .Doc(R"doc(
+Converts a flat index or array of flat indices into a tuple of coordinate
+arrays.
+
+indices: An 0-D or 1-D `int` Tensor whose elements are indices into the
+  flattened version of an array of dimensions dims.
+dims: An 1-D `int` Tensor. The shape of the array to use for unraveling
+  indices.
+output: An 2-D (or 1-D if indices is 0-D) tensor where each row has the
+  same shape as the indices array.
+
+@compatibility(numpy)
+Equivalent to np.unravel_index
+@end_compatibility
+)doc");
+
```

Comment:
The @compatibilty()... section should be before the docs for the arguments.  Arg docs must be last.

Reply:
Thanks. Done.